exercises/isogram/isogram_test.go

```diff
@@ -0,0 +1,50 @@
+package isogram
+
+import "testing"
+
+var testCases = []struct {
+	word     string
+	expected bool
+}{
+	{"duplicates", true},
+	{"eleven", false},
+	{"subdermatoglyphic", true},
+	{"Alphabet", false},
+	{"thumbscrew-japingly", true},
+	{"Hjelmqvist-Gryb-Zock-Pfund-Wax", true},
+	{"Heizölrückstoßabdämpfung", true},
+	{"the quick brown fox", false},
+	{"Emily Jung Schwartzkopf", true},
+	{"éléphant", false},
+}
+
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Errorf("Found testVersion = %v, want %v.", testVersion, targetTestVersion)
+	}
+}
+
+func TestIsIsogram(t *testing.T) {
+	for _, c := range testCases {
+		if IsIsogram(c.word) != c.expected {
+			t.Fatalf("FAIL: Word %q, expected %v, got %v", c.word, c.expected, !c.expected)
+		}
+
+		t.Logf("PASS: Word %q", c.word)
```

Comment:
I don't think we've been using "PASS" explicitly. @petertseng what do you think about this?